libfulltext/doi/__init__.py

```diff
@@ -5,8 +5,7 @@
 from .crossref import get_crossref_fulltext
 from .datacite import get_datacite_fulltext
 
-
-def get_doi_fulltext(config, doi):
+def get_doi_fulltext(config, doi, save_stream):
     """Get the fulltext for a DOI
 
     Args:
```

Comment:
we shouldn't let the docstrings get outdated.

Reply:
Fixed.